bin/gltf-pipeline.js

```diff
@@ -17,11 +17,11 @@ if (process.argv.length < 3 || defined(argv.h) || defined(argv.help)) {
         '  -s --separate, writes out separate geometry/animation data files, shader files and textures instead of embedding them in the glTF file.\n' +
         '  -t --separateImage, write out separate textures, but embed geometry/animation data files, and shader files.\n' +
         '  -q, quantize the attributes of this model.\n' +
-        '  --ao_diffuse, bake ambient occlusion into the diffuse texture.\n' +
-        '  --ao_scene, specify which scene to bake AO for.\n' +
-        '  --ao_rayDepth, ray distance for raytraced ambient occlusion.\n' +
-        '  --ao_resolution, resolution along one dimension for each AO texture.\n' +
-        '  --ao_samples, sample count for ambient occlusion\n';
+        '  --ao.diffuse, bake ambient occlusion into the diffuse texture. Defaults to false.\n' +
```

Comment:
It doesn't need to be part of this PR, but please make sure your roadmap includes updating the README.md (including examples) before we merge `ao-master` to `master`.